app/controllers/admins/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Admins::SessionsController < Devise::SessionsController
```

Comment:
I'm confused. Why is there both an `Admins::SessionsController` (plural Admins) and an `Admin::SessionsController` (singular Admin)? Can we get rid of this one as it's all by itself in a folder and doesn't even seem to be doing anything?